Cargo.toml

```diff
@@ -45,3 +45,5 @@ vendored-openssl = ["openssl/vendored"]
 [dependencies.openssl]
 version = "0.10"
 optional = true
+
+[patch.crates-io]
```

Comment:
This can be removed

Reply:
Oopsie, too used to servo where that block was always there even when empty